### PR TITLE
Added more dependencies for Datastore tests

### PR DESCRIPTION
**Pull request by apluscs**
Left these out the first time, oops

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -44,6 +44,28 @@
       <scope>test</scope>      
     </dependency>
     
+     <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-stubs</artifactId>
+      <version>1.9.24</version>
+      <scope>test</scope>      
+    </dependency>
```
Comment (meagancheese) [inline pom.xml:52]: Indent by one space.

**Review by meagancheese — CHANGES_REQUESTED** [review]

**Review by meagancheese — APPROVED** [review]
LGTM

**Review by meagancheese — APPROVED** [review]
LGTM

**Review by GrantComm — APPROVED** [review]
LGTM

**Comment by acmcarther** [comment]
I'm LGTMing this, but how did the PR get submitted without these? The build should've been red.

**Review by acmcarther — APPROVED** [review]

**Comment by apluscs** [comment]
l think Grant either pulled from this branch or added them in himself.

**Pull request merged**